### ISSUE TITLE
Only store shadow model logits, not shadow models

### ISSUE THREAD

**Issue by johanos1** [aidotse/LeakPro#131]
# Feature
## Desired Behavior / Functionality
LeakPro crashes when we use too many shadow models. If we could instead store only the logits it would scale better.

## What Needs to Be Done
- Store the logits instead of the shadow models
- Ensure that no other attacks suffer from this

## How Can It Be Tested
Attacks run
